.env

```diff
@@ -1,2 +1,5 @@
 PORT=8030
 FLASK_DEBUG=true
+SEARCH_API_KEY=
+SEARCH_API_URL=https://www.googleapis.com/customsearch/v1
```

Comment:
I can understand making these constants, but making them environment variables implies that they're going to change, and also requires that we set them in our k8s config. But actually (with the obvious exception of `SEARCH_API_KEY`), they'll probably never change. Could you just set them directly in the app for now? If we ever came to a case where we needed to change them (e.g. for testing) we could implement a way to change them dynamically when we come to that point. (I've been linking to this a lot lately: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)